pkg/kubelet/secret/secret_manager.go

```diff
@@ -146,6 +153,39 @@ func (s *secretStore) Delete(namespace, name string) {
 	}
 }
 
+const (
+	ObjectTTLAnnotation = "node.kubernetes.io/ttl"
```

Comment:
I was wondering where should we put it so that both controller and kubelet can use it (without cyclic dependencies etc.)